h/browser/chrome/test/uri-info-test.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+var proxyquire = require('proxyquire');
+
+describe('uriInfo', function() {
+  var server;
+  var uri = 'http://example.com/example';
+  var uriInfo;
+  var serviceUrl = 'http://hypothes.is';
+
+  beforeEach(function() {
+    server = sinon.fakeServer.create({
+      autoRespond: true,
+      respondImmediately: true
+    });
+
+    sinon.stub(console, 'error');
+
+    var settingsPromise = Promise.resolve({serviceUrl: serviceUrl});
+    settingsPromise['@noCallThru'] = true;
```

Comment:
Might be worth moving this to a small helper function (eg. in a `./test/util.js` module) which wraps the input function. That way you only have to document the reason for using `@noCallThru` in one place.